src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -5765,6 +5767,197 @@ static void M()
 
         #endregion
 
+        #region Patterns
+
+        public void SyntaxOffset_Parrern()
+        {
+            var source = @"class C { int F(object o) { if (o is 33) { return 44; }  if (o is int i) { return i; } if (o is bool) { return 1; } return 0; }}";
```

Comment:
>if (o is int i) { return i; } [](start = 84, length = 29)

nit: It'd be better to move this one to the beginning of the body, so that it's easier to count the distance.